src/core/lib/iomgr/tcp_server_posix.c

```diff
@@ -70,6 +71,8 @@
 
 #define MIN_SAFE_ACCEPT_QUEUE_SIZE 100
 
+int grpc_tcp_server_posix_expand_wildcard_addr = 0;
```

Comment:
The correct thing is probably to add a mutation pass to server builder
plugins. +Yuchen Zeng <zyc@google.com> could you have a look at what would
be required to allow a plugin to mutate channel args?

On Fri, Jul 8, 2016 at 3:44 PM Dan Born <notifications@github.com> wrote:

> In src/core/lib/iomgr/tcp_server_posix.c
> <https://github.com/grpc/grpc/pull/7047#discussion_r70151482>:
>
> > @@ -70,6 +71,8 @@
> >
> >  #define MIN_SAFE_ACCEPT_QUEUE_SIZE 100
> >
> > +int grpc_tcp_server_posix_expand_wildcard_addr = 0;
>
> I've been unable to find a good place to add it. The ideal situation would
> be to have the constructor for grpc_channel_args set a default value for
> the flag based on probing local hardware, and then the caller could
> customize the flag directly if necessary. However, it's a struct with no
> constructor, e.g.,
> https://github.com/grpc/grpc/blob/master/src/cpp/server/server.cc#L295
>
> I don't want to have to set it explicitly everywhere a grpc_channel_args
> is stack or heap allocated.
>
> I need a grpc_channel_args_init (constructor, essentially) to add my code
> to, but none exists.
>
> —
> You are receiving this because you were assigned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/grpc/grpc/pull/7047/files/a3a709782500b6e158710b49e365fd99a022354c#r70151482>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AJpudcWh1vc5ryia8Iziw_RFqtRFLk0oks5qTtLXgaJpZM4I-GnO>
> .
>